server/game/cards/04.2-TE/FeralNingyo.js

```diff
@@ -1,10 +1,25 @@
 const DrawCard = require('../../drawcard.js');
 
 class FeralNingyo extends DrawCard {
-    setupCardAbilities(ability) { // eslint-disable-line no-unused-vars
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Put into play',
+            condition: () => this.game.isDuringConflict() && this.game.currentConflict.hasElement('water'),
```

Comment:
This is fine, but you can pass 'water' to isDuringConflict, i.e.
`condition: () => this.game.isDuringConflict('water')`